src/test/ui/issues/issue-45829/import-twice.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+mod foo {
+    pub struct A;
+    pub struct B;
+}
+
+use foo::{A, A};
```

Comment:
This is a good test to add. In this case it might be nice _not_ to suggest renaming, as it is more likely a duplicated import, but adding that is something best left to a follow up PR. This would also ideally affect the case where we import both `use foo` and `use foo::{ self }`.